dcpy/utils/s3.py

```diff
@@ -106,7 +106,6 @@ def client() -> S3Client:
 def list_objects(bucket: str, prefix: str) -> list:
     """Lists all objects with given prefix within bucket"""
     objects: list = []
-    prefix = _folderize(prefix)
```

Comment:
I sort of get what I was going for here, but we have utils for things within a folder, list_objects really should be a prefix